challenges/dateAdder/dateAdder.js

```diff
@@ -0,0 +1,17 @@
+function dateAdder(date, diff) {
+  let result;
+  const diffChar = diff.slice(-1);
+  const diffNum = Number(diff.slice(0, diff.length - 1));
+
+  if(diffChar === 's') result = new Date(date.setSeconds(date.getSeconds() + diffNum));
```

Comment:
nice. Just be careful because you are changing the original date (its a side effect)